### PR TITLE
Repeater-start

### PR DESCRIPTION
**Pull request by programmin1**
Repeater-start is a repeater app used by many amateur radio users to see repeaters in the area.

I believe this is a good fit for up for grabs list?

### CODE REVIEW

**Comment by ritwik12** [comment]
@programmin1 Please fix the errors

**Comment by programmin1** [comment]
Corrected name of tag

File: _data/projects/repeater-start.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  - mapping
+  - ham-radio
+  - amateur-radio
+  - python
+  - windows
+  - linux
+  - android
+upforgrabs:
+  name: help wanted
+  link: https://github.com/programmin1/Repeater-START/issues
```
Comment (ritwik12) [inline _data/projects/repeater-start.yml]: @programmin1 This should be the link of the label you have mentioned above

**Review by ritwik12 — DISMISSED** [review]

File: _data/projects/repeater-start.yml (hunk 1 above)
Comment (programmin1) [reply _data/projects/repeater-start.yml]: Those are the help wanted issues.

Comment (programmin1) [reply _data/projects/repeater-start.yml]: Okay I changed it to /label url.

**Review by shiftkey — APPROVED** [review]

**Comment by AbsolytnaiaLove** [comment]
сб, 8 жовт. 2022, 17:51 користувач Brendan Forster ***@***.***>
пише:

**Comment by AbsolytnaiaLove** [comment]
сб, 8 жовт. 2022, 23:19 користувач AbsolytnaiaLove ***@***.***>
пише:

**Pull request merged**